### DISCUSSION TITLE
从软件许可的分类理解开源许可的魅力：比较视野 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#39] [Announcements]
# 从软件许可的分类理解开源许可的魅力：比较视野 - 「开源之道」

https://opensourceway.community/posts/os-license-and-copyleft/compare-software-license-and-best-os/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

这是从果导因;
其实, 这一光谱中, 最值得挖掘和分析的只有两端的许可证:

- 石破天惊, 发明软件专利的 Bill
- 匹夫有则, 构造 GPL 的 RMS

如果不是他俩划定光谱的两极, 其它中间派根本没有作用域...

社科有所谓比较文学之类学科, 值得借用工具来挖掘, 单是文本上, 两极从文字中折射出来的创造力闪点...

**Reply by lijiangsheng1** [reply]
* 石破天惊, 发明软件专利的 Bill
* 匹夫有则, 构造 GPL 的 RMS

这些内容，正在「开源之史」的系列四中探究。敬请期待～
